clinguin_ui/ui_menu_bar.lp

```diff
@@ -1,17 +1,20 @@
+% Copyright (c) 2024 Siemens AG Oesterreich
+% SPDX-License-Identifier: MIT
+
 #const help_message = @concat(
-    "<p><i>This UI allows to create a configuration</i></p>",
+    "<p><i>This UI allows to create a configuration in an interactive manner</i></p>",
     "<h5>Add objects</h5>",
-    "<p>In the add 'Add object' section one can add a new object to the system. Only leaf classes can be added as objects. Click on + to add the </p>",
+    "<p>In the 'Add object' section one can add a new object to the system. Only leaf classes can be added as objects. Click on + to add the object.</p>",
     "<h5>Edit objects</h5>",
-    "<p>Click on the node in the configuration graph to edit the object. The lower right pannel will show the selected node and posible changes. </p>",
+    "<p>Click on the node in the configuration graph to edit the object. The lower right panel will show the selected node and posible changes. </p>",
     "<h5>Partial and full configuration</h5>",
-    "<p>The yellow button on the menu can toggle between partial and full configuration. In partial configuration objects might not have values for attributes and lowebounds might not be filled. In full configuration the system will enforce the complete configuration. </p>",
+    "<p>The yellow button on the menu can toggle between a partial and a full configuration. In a partial configuration objects might not have values for attributes and lowebounds might not be filled. In full configuration the system will enforce the complete configuration. </p>",
     "<h5>Find incrementally</h5>",
-    "<p>Click on the 'Find incrementally' button to allow the system to create all the required objects to complete de configuration. </p>",
+    "<p>Click on the 'Find incrementally' button to allow the system to create all the required objects amd to complete the configuration. </p>",
```

Comment:
Please check spelling & grammar (in all text elements)

Reply:
I went through this, hopefully I did not miss anything anymore.